src/style/mixins.scss

```diff
@@ -156,14 +154,14 @@
 
     cursor: pointer;
 
-    height: functions.pxToRem(20);
-    width: functions.pxToRem(20);
+    height: 1.25rem;
+    width: 1.25rem;
```

Comment:
I don't want to be too nitpicky, and since both commit types behave the same, you don't have to change it, I'm just commenting here for the sake of knowledge sharing 🤗:

This is a textbook example of a `refactor`—the end result, the "behaviour", is _identical_, but you've change how the code gets us there 😊

Reply:
Thanks. I thought we don't wanna mess up the change log too much for the imaginary consumers. But maybe refactor won't go there neither? or?

Reply:
Exactly, refactor also doesn't go there, because it's not something any consumer needs to know about 👍

Reply:
Since I was fiddling around anyway, I changed the commits from `chore` to `refactor`.